vernac/attributes.ml

```diff
@@ -161,7 +220,10 @@ let () = let open Goptions in
 let program =
   enable_attribute ~key:"program" ~default:(fun () -> !program_mode)
 
-let locality = bool_attribute ~name:"Locality" ~on:"local" ~off:"global"
+let locality =
+  deprecated_bool_attribute
+    ~name:"Locality"
+    ~on:"local" ~off:"global"
```

Comment:
To me this is not a `bool_attribute` (and thus it is not a `deprecated_bool_attribute` either). `local` / `global` are just two possible locality values, but `export` is a third one that is supported by a couple of commands. Even `local` / `global` are not supported exactly by the same set of commands.

Reply:
Indeed note the redundancy already in place; I'd say that unifying this is a bit orthogonal to this PR, and may involve actually modifying the underlying data type so 2 valued localities are different from 3 valued ones?

Reply:
Improved.